### DISCUSSION TITLE
How to hide the STATS from the report?

### DISCUSSION THREAD

**Discussion by kiragengis** [allure-framework/allure2#2456] [Questions & Support]
I want to hide these stats:

![image](https://github.com/allure-framework/allure2/assets/33094881/40a37d4b-a2d4-4f77-a137-6af035ce3bb3)

Is this possible?

**Comment by baev** [comment]
Could you please give us a bit more context why do you need this?

**Reply by kiragengis** [reply]
Hi @baev, thank you for answering. There is no technical reason. The goal is to have cleaner information on the screen. My team is not going to use that information.

**Reply by noomorph** [reply]
Use custom css, maybe?

**Reply by kiragengis** [reply]
It sounds like I would have to do this through CSS. I expected some flags in the code to enable or disable certain UI components/stats. I am going to close the discussion.
